packages/dev/core/src/AudioV2/abstractAudio/mainAudioBus.ts

```diff
@@ -0,0 +1,23 @@
+import { AbstractAudioBus } from "./abstractAudioBus";
+import type { AudioEngineV2 } from "./audioEngineV2";
+import type { IVolumeAudioOptions } from "./subNodes/volumeAudioSubNode";
+
+/** */
+export interface IMainAudioBusOptions extends IVolumeAudioOptions {}
+
+/**
+ * Abstract class representing a main audio bus.
+ *
+ * Main audio buses are the last bus in the audio graph.
+ *
+ * Unlike {@link AudioBus} instances, `MainAudioBus` instances have no spatial audio and stereo output capabilities,
+ * and they cannot be connected downstream to another audio bus. They only connect downstream to the audio engine's
+ * main output.
+ *
+ * Main audio buses are created by the {@link CreateMainAudioBusAsync} function.
+ */
+export abstract class MainAudioBus extends AbstractAudioBus {
```

Comment:
required for a future implementation? currently there is no difference between this and an AbstractAudioBus (and apologies if you have already explained this)

Reply:
There will be a `SecondaryAudioBus` coming later.